appengine-java11/README.md

```diff
@@ -0,0 +1,67 @@
+# Google App Engine Standard Environment Samples for Java 11
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/ludoch/samples&page=editor&open_in_editor=appengine-java11/README.md">
+<img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
+
+This is a repository that contains Java code samples for [Google App Engine][ae-docs]
+standard Java 11 environment.
+
+[ae-docs]: https://cloud.google.com/appengine/docs/java/
+
+## Prerequisites
+
+**Private Alpha**
+To access the Java 11 Google App Engine standard runtime, you'll need to apply
+to become part of the App Engine [Alpha program](https://docs.google.com/forms/d/e/1FAIpQLSf5uE5eknJjFEmcVBI6sMitBU0QQ1LX_J7VrA_OTQabo6EEEw/viewform).
+
+
+### Download Maven
+
+These samples use the [Apache Maven][maven] build system. Before getting
+started, be sure to [download][maven-download] and [install][maven-install] it.
+When you use Maven as described here, it will automatically download the needed
+client libraries.
+
+[maven]: https://maven.apache.org
+[maven-download]: https://maven.apache.org/download.cgi
+[maven-install]: https://maven.apache.org/install.html
+
+### Create a Project in the Google Cloud Platform Console
+
+If you haven't already created a project, create one now. Projects enable you to
+manage all Google Cloud Platform resources for your app, including deployment,
+access control, billing, and services.
+
+1. Open the [Cloud Platform Console][cloud-console].
+1. In the drop-down menu at the top, select **Create a project**.
+1. Give your project a name.
+1. Make a note of the project ID, which might be different from the project
+   name. The project ID is used in commands and in configurations.
+
+[cloud-console]: https://console.cloud.google.com/
+
+### Google Cloud Shell, Open JDK 11 setup:
+
+To switch to an Open JDK 11 in a Cloud shell session, you can use:
+
+```
+   sudo update-alternatives --config java
+   # And select the usr/lib/jvm/java-11-openjdk-amd64/bin/java version.
+   # Also, set the JAVA_HOME variable for Maven to pick the correct JDK:
+   export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+```
+
+## Set up
+Build the samples with maven:
+```
+mvn clean package
+```
+
+### appengine-simple-jetty-main
```

Comment:
All of these samples could also be used w/ Cloud Run w/ minor modifications.  You might consider releasing them both ways.